docs/tracking.md

```diff
@@ -1 +1,1030 @@
+aligned with the track momentum. By using Jacobians to the curvilinear frame
+and the corresponding inverse matrices, conversions between any two
+coordinate systems can be performed.
+
+The second part is the calculation of the evolution of the covariance matrix
+during the propagation between surfaces. To this end, a semi-analytical
+method which calculates the effective derivatives between two consecutive
+RKN steps can be used. By accumulating the Jacobian
+matrices calculated for each step, the effective Jacobian between the
+starting point and the destination can be obtained.
+
+## Material effects
+
+Charged particles interact with matter as they pass through it. Since
+particle detectors inevitably consist of some form or material, this effect
+cannot be completely avoided. By building tracking detectors as light as
+possible, and arranging passive components, such as services and support
+structures carefully, the material a particle encounters before being
+measured can be reduced. Charged particles traversing any form of matter
+undergo elastic and inelastic interactions with the atomic structure of the
+material, depending on the particle properties.
+
+(multiple_scattering)=
+:::{figure} /figures/tracking/multiple_scattering.svg
+:width: 200px
+:align: center
+Illustration of the effect of multiple scattering on the
+trajectory of a charged particle passing through a block of material.
+Entering from the left, it undergoes a series of scattering events,
+deflecting the trajectory statistically, before exiting on the right.
+:::
+
+In elastic interactions, the particle does not lose a significant amount of
+energy, while its trajectory is affected. {numref}`multiple_scattering` shows a
+sketch of the way multiple Coulomb scattering affects the direction of a
+particle trajectory. In addition, a shift in the transverse plane relative to
+the incident direction can occur. As the scattering events occur in
+statistically independent directions, the means of both the deflection and
+offset tends toward zero as the number of scatters increases. Therefore, in the
+numerical particle propagation, this can be accounted for by simply increasing
+the uncertainties associated with the direction, depending on the amount of
+material encountered.
+
+On the other hand, there are interactions during which the particle loses
+some of its energy. Relevant processes here are ionization, as well as
+bremsstrahlung for light particles like electrons. For hadronic particles,
+hadronic interactions with the nuclei of surrounding material is another
+process of interest. In such hadronic interactions, the incoming particle
+often disintegrates, and does not propagate further. Since the size of
+ionization losses only fluctuates to a small degree for thin layers of
+material, they can usually be accounted for by reducing the trajectory energy
+correspondingly. For bremsstrahlung, where fluctuations are much larger,
```

Comment:
(and the effect can in addition not being modelled with a Gaussian distribution adequately)